index.html

```diff
@@ -1,45 +1,56 @@
 <!doctype html>
-<html lang="en" data-framework="es6">
+<html lang="en" data-framework="javascript">
 	<head>
 		<meta charset="utf-8">
-		<title>Vanilla ES6 • TodoMVC</title>
+		<title>VanillaJS • TodoMVC</title>
+		<link rel="stylesheet" href="node_modules/todomvc-common/base.css">
 		<link rel="stylesheet" href="node_modules/todomvc-app-css/index.css">
+		<style>
+		 .bold {
+			 font-weight: 700;
+		 }
+		</style>
```

Comment:
Is this necessary? (I didn't see it in the python PR that uses the `.bold` class)

Reply:
...whoops. 🤦🏼 Get ready for another PR

Reply:
Can we also fix the mixed indentation / make it be a single line without returns?

Or we can have the guide add `style="font-weight:bold"` instead of a class? That way they can also add any styles they want and the guide can suggest bold.